copilot/plugin/v2.3/schema.json

```diff
@@ -457,7 +455,17 @@
             "properties": {
                 "local_endpoint": {
                     "type": "string",
```

Comment:
We need to enforce the value of `local_endpoint` to `Microsoft.Office.Addin`

Reply:
However, it may take on more values in the future

Reply:
This is now updated.